.travis.sh

```diff
@@ -3,7 +3,14 @@
 set -e
 
 if [[ $BUILD == dub ]]; then
-    dub test
+    if [[ -n $LIBDPARSE_VERSION ]]; then
+        rdmd ./d-test-utils/test_with_package.d $LIBDPARSE_VERSION libdparse -- dub test
+    elif if [[ -n $DSYMBOL_VERSION ]]; then
```

Comment:
I'm pretty sure the `if` should not be here.

Reply:
fixed